pages/blog/beyond-boundaries.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Beyond Boundaries - Empowering Through OSS Mentorship'
+date: 2023-07-12T06:00:00+01:00
+type: Community
+tags:
+  - Mentorships
+  - OSS
+  - AsyncAPI
+cover: /img/posts/beyond.webp
+weight: 10
+authors:
+  - name: Azeez Elegbede
+    photo: /img/avatars/ace.webp
+    link: https://twitter.com/_acebuild
+    byline: AsyncAPI Preacher
+---
+
+In 2022 we piloted our own very innovative mentorship program that focuses on open-source software(OSS). We envisioned a platform where aspiring developers could learn, grow, and contribute to exciting projects while receiving guidance from experienced mentors.
+
+## Our Backstory
+
+At AsyncAPI, we value giving back to the community, as it gives us a sense of purpose as a community-first organization, and we are always looking for ways to participate in Opensource mentorship programs like Google Summer of Code(GSoC), Google Season of Docs(GSoD) and more because the fulfilling feeling of giving back and contributing to society gotten from this programs is unparalleled.
+
+As an open-source organization, we participate so that we can collaborate with a broader reach of open-source enthusiasts and help bring more folks into the open-source world(which we are already doing). Furthermore, our reason for participating in these programs comes from our passion for open source and the care to support and encourage open-source contributors by building and participating in mentorships.
+
+## Why Start Our Own OSS Mentorship Program?
+
+In 2021, AsyncAPI participated in its first paid mentorship program alongside Postman under Google Summer Of Code(GSoC). After applying to participate as an organization for GSoC in 2021, we got rejected and were opportune to participate alongside Postman.
+
+Postman is an established software development company known for its commitment to open-source initiatives. We received a huge ton of engagement from folks interested in our projects despite having just a few slots allocated to us by Postman.
+
+Fast forward to 2022, we applied for GSoC and we got rejected again this time, Postman also got rejected and this left us in shock and disappointment because a lot of folks were already looking forward to joining us. The shock and disappointment permeated the team's spirits but they refused to let this setback deter them from their mission. 
```

Comment:
```suggestion
Fast forward to 2022, we applied for GSoC and got rejected again. This time, Postman also got rejected, leaving us in shock and disappointed because many folks were already looking forward to joining us. The shock and disappointment permeated the team's spirits, but they refused to let this setback deter them from their mission. 
```